elements/lisk-p2p/test/integration/custom_node_info.ts

```diff
@@ -12,60 +12,59 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-import { P2P } from '../../src/index';
-import { createNetwork, destroyNetwork } from '../utils/network_setup';
+// import { P2P } from '../../src/index';
+// import { createNetwork, destroyNetwork } from '../utils/network_setup';
+
+// TODO: Skipping as strict schema doesn't allow custom fields. Enable this test when users will be allowed to pass custom schema.
```

Comment:
Instead of commenting out, you can simply .skip the test

Reply:
If I skip eslint throws an error and even with `eslint-disable` it's not working. It works on Mocha